src/activities/assignments/AssignmentEntity.js

```diff
@@ -780,6 +798,12 @@ export class AssignmentEntity extends Entity {
 			}
 		}
 
+		if (typeof assignment.defaultScoringRubricId !== 'undefined' &&
+			assignment.defaultScoringRubricId !== this.getDefaultScoringRubric() &&
+			this.canEditDefaultScoringRubric()) {
+			fields.push({ name: 'defaultScoringRubricId', value: assignment.defaultScoringRubricId });
```

Comment:
So in the assignment store I set it to a default value if there is none provided when we retrieve the id, I don't set it to a default when saving because the server does accept `null` as a value for this field.